go.work

```diff
@@ -0,0 +1,7 @@
+go 1.22.0
```

Comment:
go.mod has version 1.21.0 Do we need them both in same version?

Reply:
updated 👍